core/src/components/textarea/textarea.scss

```diff
@@ -323,6 +323,8 @@
   min-height: inherit;
   max-height: inherit;
 
+  grid-auto-rows: 100%;
```

Comment:
Can we add a comment noting that this is to fix a Webkit bug, and/or include a tech debt ticket to remove it once the bug is fixed?

Reply:
Done! https://github.com/ionic-team/ionic-framework/pull/27766/commits/d8c1b171084effaf2a6d662b74e94e386f00cbe8

Reply:
I agree with this comment.